jobs/clamav/templates/bin/pre-start.erb

```diff
@@ -2,6 +2,21 @@
 
 export MINUTE=$(expr $RANDOM % 60)
 export HOUR=$(shuf -i <%= p('clamav.nightly.scan.start') %>-<%= p('clamav.nightly.scan.stop') %> -n 1)
+export CLAMAV_HOME=/var/vcap/packages/clamav
+export DATADB_DIR=/var/vcap/data/clamav/database
+
+# make database DIR if missing
+if [ ! -d "${DATADB_DIR}" ]; then
+	mkdir -p ${DATADB_DIR}
+fi
+
+# check if there is an existing DB in data
+if [ -z "$(ls -A -- "${DATADB_DIR}")" ]; then
```

Comment:
do we not know the name of the database?

Reply:
@bengerman13 this was to check if the DB directory path existed and had any contents.  If the folder was blank, then trigger an update-db to build a new db otherwise let the normal freshclam process up the database on it's schedule.

Reply:
right - I mean we're `ls`ing the dir, which seems odd given we're probably concerned with one specific file:

```
if [[ ! -e "${DATADB_DIR}/clamav.db"]]; then
```

probably works out the same in most/all cases, just seems like a roundabout way to check to me